applications/GeoMechanicsApplication/custom_conditions/Pw_normal_flux_condition.hpp

```diff
@@ -37,13 +37,13 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) PwNormalFluxCondition : public PwCon
 
     KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION( PwNormalFluxCondition );
     
-    typedef std::size_t IndexType;
-	typedef Properties PropertiesType;
-    typedef Node <3> NodeType;
-    typedef Geometry<NodeType> GeometryType;
-    typedef Geometry<NodeType>::PointsArrayType NodesArrayType;
-    typedef Vector VectorType;
-    typedef Matrix MatrixType;
+    using IndexType = std::size_t;
+    using PropertiesType = Properties;
+    using NodeType = Node<3>;
+    using GeometryType = Geometry<NodeType>;
+    using NodesArrayType = Geometry<NodeType>::PointsArrayType;
```

Comment:
You may be able to simplify this even further:
```suggestion
    using NodesArrayType = GeometryType::PointsArrayType;
```
But please, don't take my word for it. Test it first!

If you agree and assuming it works, don't forget to check the other files for the same thing. Thanks.